resources/views/components/form/genres.blade.php

```diff
@@ -0,0 +1,32 @@
+<?php
+use Illuminate\Support\Arr;
+/**
+ * We want to pre-load the data from the model, or what has been sent with the form.
+ */
+$selectedOption = [];
+
+$model = Arr::get($options, 'model');
+
+// Try to load what was sent with the form first, in case there was a form validation error
+$previous = old('genres[]');
+$fieldUniqIdentifier = 'genres_' . uniqid();
+
+if (!empty($previous)) {
+    //dd($previous);
+}
+// If we didn't get anything, and there is a model sent, use that
+elseif(!empty($model)) {
+    /** @var \App\Models\Genre $genre */
+    foreach ($model->genres as $genre) {
+        $selectedOption[$genre->id] = __('genres.' . $genre->slug);
+    }
+}
+?>
+<label>{{ __('entities.families') }}</label>
+
+<select multiple="multiple" name="genres[]" class="form-control select2 join-item" data-tags="true" style="width: 100%" data-url="{{ route('genres.find') }}" data-allow-clear="true" data-allow-new="false" data-placeholder="" id="{{ $fieldUniqIdentifier }}">
```

Comment:
I'm not seeing where it's being limited to max 3 genres. It should also display all 20 options in the dropdown at all times because users won't be able to guess what are the available options. Instead of loading them via ajax, we can enable them directly in the page.